safe-room.lic

```diff
@@ -47,6 +57,14 @@ class SafeRoom
     give_and_take(settings.safe_room_id, settings.safe_room_give, settings.safe_room_take)
   end
 
+  def get_health_score(health_hash)
```

Comment:
You could replace this method with `health_hash.map { |k, v| k * v.count }.reduce(:+)`